generators/server/templates/src/main/java/package/config/_OAuth2ServerConfiguration.java

```diff
@@ -70,40 +97,48 @@ public void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/swagger-resources/configuration/ui").permitAll()
                 .antMatchers("/swagger-ui/index.html").hasAuthority(AuthoritiesConstants.ADMIN);
         }
+<% if (databaseType == 'sql') { %>
+        @Override
+        public void configure(ResourceServerSecurityConfigurer resources) throws Exception {
+            resources.resourceId("res_<%= baseName %>").tokenStore(tokenStore);
+        }<% } %>
     }
 
     @Configuration
     @EnableAuthorizationServer
     protected static class AuthorizationServerConfiguration extends AuthorizationServerConfigurerAdapter {<% if (databaseType == 'sql') { %>
 
         @Inject
-        private DataSource dataSource;<% } %><% if (databaseType == 'mongodb') { %>
-
-        @Inject
-        private OAuth2AccessTokenRepository oAuth2AccessTokenRepository;
-
-        @Inject
-        private OAuth2RefreshTokenRepository oAuth2RefreshTokenRepository;<% } %>
-
+        private DataSource dataSource;<% } %>
+<% if (databaseType != 'sql') { %>
         @Inject
         private JHipsterProperties jHipsterProperties;
-
+}<% } %>
```

Comment:
sorry about that mistake, I got lost in the sql&jwt&mongo yesterday.
thanks for your patient reviewing.